packages/client/lib/client/index.ts

```diff
@@ -163,6 +167,9 @@ export default class RedisClient<
   static #createModuleCommand(command: Command, resp: RespVersions) {
     const transformReply = getTransformReply(command, resp);
     return async function (this: NamespaceProxyClient, ...args: Array<unknown>) {
+      if (command.unstableResp3Module && resp == 3 && !this._self._self.#options?.unstableResp3Modules) {
+        throw new Error("unstable resp3 module, client not configured with proper flag");
```

Comment:
need to update the message + include a link

Reply:
agreed, but unsure what it should be at the moment.

Reply:
as discussed on zoom, moved this to how the command is actually attached to object in attachCommand

Reply:
added a FIXME to it, to know we have to work on that string